common/reactive/src/test/java/io/helidon/common/reactive/jmh/ShakespearePlaysScrabble.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.reactive.jmh;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.openjdk.jmh.annotations.*;
+
+/**
+ * Original copyright Jose Paumard, 2019.
+ * https://github.com/JosePaumard/jdk8-stream-rx-comparison-reloaded
+ */
+@State(Scope.Benchmark)
+public class ShakespearePlaysScrabble {
```

Comment:
Just to be sure, can I ask @barchetta if its ok to reuse Apache 2.0 licenced code here? Thx a lot

Reply:
Yes. OK.